package.json

```diff
@@ -18,23 +18,28 @@
   "private": true,
   "devDependencies": {
     "@types/chai": "^4.3.8",
+    "@types/markdown-it": "^13.0.6",
     "@types/mocha": "^10.0.2",
     "@types/node": "^18.17",
     "@types/vscode": "^1.83.0",
+    "@types/vscode-webview": "^1.57.3",
     "@typescript-eslint/eslint-plugin": "^6.8.0",
     "@typescript-eslint/parser": "^6.8.0",
     "@vscode/debugadapter": "^1.63.0",
     "@vscode/debugprotocol": "^1.63.0",
     "@vscode/extension-telemetry": "^0.8.5",
+    "@vscode/markdown-it-katex": "^1.0.0",
```

Comment:
The webview uses markdown-it and katex to align more closely with how VS Code renders Notebooks and LaTeX. These packages are also faster and smaller. We can consider if the playground should move also at a later time.